src/swift/SwiftAPIViewCore/Sources/Models/ExtensionModel.swift

```diff
@@ -177,9 +195,9 @@ extension Array<ExtensionModel> {
                     }
                 }
             } else {
-                resolved[ext.definitionId] = ext
+                resolved[defId] = ext
             }
         }
-        return Array(resolved.values).sorted(by: {$0.definitionId < $1.definitionId })
+        return Array(resolved.values).sorted(by: {$0.definitionId! < $1.definitionId! })
```

Comment:
Could these ever be null at this point or is a guard needed?

Reply:
There's a guard on line 185-187 that guarantees there will be a value.